trivy-image-scan/action.yml

```diff
@@ -53,6 +53,13 @@ runs:
       run: |
         echo "FORMAT=sarif" >> $GITHUB_ENV
         echo "OUTPUT=''trivy-results.sarif''" >> $GITHUB_ENV
+    
+    - name: Set up output values
+      if: ${{ inputs.output-mode == 'json' }}
+      shell: bash
+      run: |
+        echo "FORMAT=json" >> $GITHUB_ENV
+        echo "OUTPUT=trivy-results.json" >> $GITHUB_ENV
```

Comment:
looks fine but you didn't start from a clean branch, please resolve conflicts first

Reply:
resolved the conflicts